src/app/browse/br-clothing/br-clothing.component.ts

```diff
@@ -1,16 +1,49 @@
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
+import { item } from '../../Item';
+import { ItemService } from '../../item.service';
+import { Mock } from '../../MockBook';
 
 @Component({
   selector: 'app-br-clothing',
   templateUrl: './br-clothing.component.html',
   styleUrls: ['./br-clothing.component.css']
 })
 export class BrClothingComponent implements OnInit {
+  retrievedItems: item[];
+  items: item[];
 
-  constructor(public router: Router) { }
 
+  constructor(public router: Router, 
+             private ItemService: ItemService) { }
+
+
+  getItems():void{
+  	this.ItemService.getItems()
+  		.subscribe(items => this.retrievedItems = items)
+
```

Comment:
Also, `.subscribe()` registers a callback function to the Observable that is fired every time a new value is emitted. This means that you cannot count on the value of `this.retrievedItems` being populated before subsequent lines are executed.